src/middlewares/validationMiddleware.ts

```diff
@@ -25,16 +25,10 @@ export const validateUserBody = (req: Request, res: Response, next: NextFunction
 };
 
 export const validateUserIdQuery = (req: Request, res: Response, next: NextFunction) => {
```

Comment:
where are we using this middleware?

Reply:
This middleware is being used to see if the ID in the endpoint is either missing or invalid.